samples/msal-react-samples/react-router-sample/src/App.js

```diff
@@ -0,0 +1,48 @@
+import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
+// Material-UI imports
+import { ThemeProvider } from '@material-ui/core/styles';
+import Grid from "@material-ui/core/Grid";
+import { theme } from "./styles/theme";
+
+// MSAL imports
+import { PublicClientApplication } from "@azure/msal-browser";
+import { MsalProvider } from "@azure/msal-react";
+import { msalConfig } from "./authConfig";
+
+// Sample app imports
+import { PageLayout } from "./ui-components/PageLayout";
+import { Home } from "./pages/Home";
+import { Profile } from "./pages/Profile";
+
+function App() {
+  const msalInstance = new PublicClientApplication(msalConfig);
```

Comment:
I think you will want to put this outside the `App` function, just in case the `App` component rerenders.

Reply:
Good call, moved. Thanks!